framework/src/modules/interoperability/mainchain/types.ts

```diff
@@ -0,0 +1,7 @@
+export enum ForwardResult {
```

Comment:
maybe `ForwardCCMsgResult` is sounding better?

Reply:
Done